experimental/dds/tree2/src/feature-libraries/schema-index/schemaSummarizer.ts

```diff
@@ -35,31 +36,44 @@ import { isJsonObject, JsonCompatible, JsonCompatibleReadOnly } from "../../util
 import { Format } from "./format";
 import { encodeRepo, makeSchemaCodec } from "./codec";
 
-/**
- * The storage key for the blob in the summary containing schema data
- */
-const schemaBlobKey = "SchemaBlob";
-
 const schemaStringKey = "SchemaString";
 
+/**
+ * Manages the collaboration window for incremental summarization.
+ */
+export interface CollabWindow {
```

Comment:
This feels like a type that deserves to be in its own file (in feature libraries) as it will probably be reused for any future incremental summarization we implement.

Reply:
Moved interface to separate file `incrementalSummarizationUtils.ts` in feature libraries